tests/unit/lms/services/oauth2_token_test.py

```diff
@@ -79,6 +105,10 @@ def svc(self, pyramid_request):
             pyramid_request.lti_user.user_id,
         )
 
+    @pytest.fixture
+    def logger(self, patch):
+        return patch("lms.services.oauth2_token.logger")
```

Comment:
You could use pytest's builtin `caplog` fixture instead of patching the logger: https://docs.pytest.org/en/stable/logging.html?highlight=caplog#caplog-fixture